net-p2p/rtorrent/rtorrent-0.9.6.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="BitTorrent Client using libtorrent"
+DESCRIPTION="rTorrent is a text-based ncurses BitTorrent client written in C++, \
+based on the libTorrent libraries for Unix, whose author's goal is 'a \
+focus on high performance and good code'."
+HOMEPAGE="http://libtorrent.rakshasa.no/"
+COPYRIGHT="2005-2016 Jari Sundell"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/rakshasa/rtorrent/archive/$portVersion.zip"
+CHECKSUM_SHA256="228cd2fac94860dcee42a03f4bdc6a4deeec9b33ed7de356b64fc5e175481ee8"
+PATCHES="rtorrent_x86-$portVersion.patchset"
+
+ARCHITECTURES="x86 ?x86_gcc2 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	rtorrent$secondaryArchSuffix = $portVersion
+	cmd:rtorrent$secondaryArchSuffix = $portVersion compat >= 0.13
```

Comment:
Is this OK to have **`compat >= 0.13`** when **`$portVersion`** is **`0.9.6`**?
Wouldn't it be better to simply drop the whole **`compat >= 0.13`**?